content/iOS/RSFormView/RSFormViewExample.md

```diff
@@ -0,0 +1,310 @@
+The naming should be self explanatory here, and each configure function should be pretty straight forward except for  `configureConstraints` were all the autolayout magic is gonna happen.
+
+``` swift
+func configureSubmitButton() {
+  updateSubmitButton(enabled: false)
+  submitButton.layer.cornerRadius = 8
+  submitButton.setTitle("SUBMIT", for: .normal)
+  submitButton.setTitleColor(UIColor.white, for: .normal)
+  submitButton.translatesAutoresizingMaskIntoConstraints = false
+  submitButton.addTarget(self, action: #selector(submitButtonTapped), for: .touchUpInside)
+
+  view.addSubview(submitButton)
+}
+
+func configureLabels() {
+  descriptionLabel.numberOfLines = 0
+  descriptionLabel.lineBreakMode = .byWordWrapping
+  descriptionLabel.translatesAutoresizingMaskIntoConstraints = false
+  descriptionLabel.textColor = UIColor.brightGray
+  descriptionLabel.textAlignment = .center
+  descriptionLabel.text = "This is an example of a RSFormView changing its aspect on an user triggered event"
+  descriptionLabel.font = UIFont.systemFont(ofSize: 22,
+  weight: .semibold)
+
+  view.addSubview(descriptionLabel)
+
+  switchLabel.textColor = UIColor.brightGray
+  switchLabel.textAlignment = .left
+  switchLabel.translatesAutoresizingMaskIntoConstraints = false
+  switchLabel.text = "Enable dark mode"
+  switchLabel.font = UIFont.systemFont(ofSize: 19)
+
+  view.addSubview(switchLabel)
+}
+
+func configureSwitch() {
+  modeSwitch.isOn = false
+  modeSwitch.onTintColor = UIColor.dodgerBlue
+  modeSwitch.translatesAutoresizingMaskIntoConstraints = false
+  modeSwitch.addTarget(self, action: #selector(switchValueChanged), for: .valueChanged)
+
+  view.addSubview(modeSwitch)
+}
+```
+
+In each of this functions we are setting the initial state for each UI Component (except the formView we are leaving that for the end).
+We are setting titles to labels and buttons, as well as background colors. 
+We are setting actions for the submit button and the `modeSwitch` and setting `translatesAutoresizingMaskIntoConstraints`  to false to every UIComponent, since we are relying on autolayout to place the views.
+Finally we are adding the views to our viewController's view. 
+You are going to be seeing compiler errors when setting submitButton and modeSwitch targets since the referenced selectors are not defined yet, ignore those for now. 
+
+Now lets configure the formView: 
+
+``` swift
+func configureFormView() {
+  formView.delegate = self
+  formView.viewModel = formHelper
+  formView.translatesAutoresizingMaskIntoConstraints = false
+
+  view.addSubview(formView)
+}
+```
+
+You should be seeing two compiler errors in this function. 
+This is because your view controller doesn't comply to `FormViewModel` delegate and  `formHelper` was never defined. 
+We are gonna see how to do this in the next section. 
+
+Now lets tackle  `configureConstraints` the responsible of setting all the layouting rules for every view in our view controller.
```

Comment:
'the' is not needed: "Now lets tackle  `configureConstraints`, responsible of setting all the layouting rules for every view in our view controller"